prow/cmd/deck/static/script.js

```diff
@@ -645,7 +648,10 @@ function prRevisionCell(build) {
 
 function drawJobBar(total, jobCountMap) {
   const states = ["success", "pending", "triggered", "error", "failure", "aborted", ""];
-  states.forEach(state => {
+  states.sort((s1, s2) => {
+    return jobCountMap.get(s1) - jobCountMap.get(s2);
+  });
```

Comment:
nit: It might make more sense to keep the states in a consistent order instead of sorting them by number of jobs in that state. Feel free to ignore me.

Reply:
The state order is always consistent because the html tags are unchanged. It’s just allocating width from smallest to biggest states.

Reply:
Maybe I put a comment there